tools/cockpit.spec

```diff
@@ -42,7 +42,7 @@
 %define build_optional 1
 
 # cockpit's firewall service definition moved to firewalld
-%if 0%{?fedora} >= 29 || 0%{?rhel} >= 8
+%if 0%{?fedora} >= 29 || (0%{?rhel} == 7 && "%{os_version_id}" == "7.7") || 0%{?rhel} >= 8
```

Comment:
This would better be `"%{os_version_id}" != "7.6"` to still be correct for 7.8 and beyond.